docs/docs/quickstart.md

```diff
@@ -0,0 +1,27 @@
+---
+id: quickstart
+title: Quick Start
+sidebar_label: Quick Start
+---
+
+1.  Install [the GitHub app](https://github.com/marketplace/kodiakhq#pricing-and-setup)
+
+2.  Create a `.kodiak.toml` file in the root of your repository with the following contents
+
+    ```toml
+    # .kodiak.toml
+    # Minimal config. version is the only required field.
+    version = 1
+    ```
+
+3.  [Configure GitHub branch protection](https://help.github.com/en/articles/configuring-protected-branches)
+
+4.  Create an automerge label (default: "automerge")
+
+5.  Start auto merging PRs with Kodiak
+
+    Label your PRs with your `automerge` label and let Kodiak do the rest! 🎉
+
+See the [docs](<docUrl("quickstart.html")>) for additional setup information.
```

Comment:
This doesn't seem to render correctly.

I get this link: https://deploy-preview-234--suspicious-galileo-259456.netlify.com/docs/docUrl(%22quickstart.html%22)